src/Hl7.Fhir.R5.Core/Serialization/FhirJsonParser.cs

```diff
@@ -58,14 +58,14 @@ public async Tasks.Task<Base> ParseAsync(string json, Type dataType = null)
         /// <inheritdoc cref="ParseAsync(JsonReader, Type)" />
         public Base Parse(JsonReader reader, Type dataType = null)
         {
-            var rootName = dataType != null ? ModelInfoNEW.GetFhirTypeNameForType(dataType) : null;
+            var rootName = dataType != null ? ModelInfo.ModelInspector.GetFhirTypeNameForType(dataType) : null;
```

Comment:
In the satellites, saying `ModelInfo.ModelInspector.X()` is the same as `ModelInfo.X()`, right?

Reply:
Yes, it is. I changed it.